api/lib/domain/models/UserLogin.js

```diff
@@ -25,35 +25,35 @@ class UserLogin {
     this.temporaryBlockedUntil = null;
   }
 
-  shouldBlockUserTemporarily() {
+  shouldMarkUserAsTemporarilyBlocked() {
     return this.failureCount % settings.login.temporaryBlockingThresholdFailureCount === 0;
   }
 
-  blockUserTemporarily() {
+  markUserAsTemporarilyBlocked() {
     const commonRatio = Math.pow(2, this.failureCount / settings.login.temporaryBlockingThresholdFailureCount - 1);
     this.temporaryBlockedUntil = new Date(Date.now() + settings.login.temporaryBlockingBaseTimeMs * commonRatio);
   }
 
-  hasBeenTemporaryBlocked() {
+  hasBeenMarkedAsTemporaryBlocked() {
     return this.failureCount > 0 || !!this.temporaryBlockedUntil;
```

Comment:
Si on veut reprendre le même concept que pour la version `blocked`

```suggestion
    return !!this.temporaryBlockedUntil;
```

Reply:
En fait, ce n'est pas l'équivalent de `isUserMarkedAsBlocked` pour savoir s'il faut bloquer l'utilisateur dans le préhandler, c'est une méthode qui est appelée pour savoir s'il faut réinitialiser le `failureCount` quand un bon mot de passe est entré. Je vais la renommer `hasFailedAtLeastOnce` pour que ce soit plus clair.